security/pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.devonfw.tools.IDEasy.dev</groupId>
+        <artifactId>ide</artifactId>
+        <version>dev-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.example</groupId>
```

Comment:
```suggestion
```
Copy&Paste from the web? If you omit it here, you inherit out `groupId` from `<parent>` what fully makes sense.

Reply:
Okey. Yes I did:)

Reply:
Looks like this was fixed already. Please resolve.

Reply:
Removed now.